backend/app/contact/routes.py

```diff
@@ -14,3 +14,55 @@ def delete_contact_type(id):
     db.session.commit()
 
     return jsonify(contact.serialize)
+
+@contact.route("/<uuid:id>/edit", methods=["PUT"])
```

Comment:
@dgiltz Was wondering whether you could change the route name to `"/<uuid:id>"`? Feel free to checkout our discussion on #99.